.ci.yaml

```diff
@@ -2796,6 +2807,28 @@ targets:
         ["devicelab", "android", "linux"]
       task_name: new_gallery__transition_perf
 
+  # Mokey, Impeller
+  - name: Linux_mokey new_gallery_impeller__transition_perf
+    recipe: devicelab/devicelab_drone
+    bringup: true # Device exists only in staging.
+    presubmit: false
+    timeout: 60
+    properties:
+      tags: >
+        ["devicelab", "android", "linux", "mokey"]
```

Comment:
Looks like the tags are going to be the same for all linux_mokey builds ? Can they live in the linux_mokey section?

Reply:
I don't really know how the .ci.yaml file is supposed to work, and I'm just following suit with every other build. If putting the common tags under the `linux_mokey` section is expected to work, I can try that cleanup for the whole file in a separate PR.